tests/test_detectors.py

```diff
@@ -92,3 +92,13 @@ def test_hampel_detector(data_series):
     # Validate if the found anomalies are also in the expected anomaly set
     # NB Not necessarily all of them
     assert all(i in expected_anomalies_indices for i in anomalies_indices)
+
+
+def test_hampel_numba_detector(data_series):
```

Comment:
Is there a test that the algorithm with/without numba produces identical output? It should only be

Reply:
Yes, great idea. I added that test.